hack/rpm-deps.sh

```diff
@@ -159,6 +163,16 @@ sidecar_shim="
     python3
 "
 
+cc_toolchain="
```

Comment:
Why the toolchain in the rpm? Isn't it already included in the builder container?

Reply:
The idea is to drop it from the builder container and to handle it via bazel instead. If we can get this to work, and we manage to build a working cross-compiling toolchain from RPMs, then we can later adopt the same approach across the board, including aarch64 and x86_64. But right now the bazel version is just a work in progress, hence why it shows up in two places :)